src/backend/base/langflow/components/jigsawstack/vocr.py

```
@@ -0,0 +1,107 @@
+
+    outputs = [
+        Output(display_name="VOCR results", name="vocr_results", method="vocr"),
+    ]
+
+    def vocr(self) -> Data:
+        try:
+            from jigsawstack import JigsawStack
+        except ImportError as e:
+            raise ImportError(
+                "JigsawStack package not found"
+            ) from e
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)